src/app/layout/footer/footer.component.html

```diff
@@ -21,145 +21,56 @@
-					<td><a href=""><i class="fab fa-twitter"></i></a></td>
-					<td><a href=""><i class="fab fa-instagram"></i></a></td>
-					<td><a href=""><i class="fab fa-youtube"></i></a></td>
+					<td><a href="https://twitter.com/DigitalTaxUSA"><i class="fab fa-twitter"></i></a></td>
+					
+					<td><a href="https://www.linkedin.com/in/digitaltaxusa/"><i class="fab fa-linkedin-in"></i></a></td>
 				</tr>
 			</table>
 		</div>
 		
 	</div>
 
 	<div class="footer_menu">
-		<div>
-			<p class="menu_head">About</p>
-			<ul>
-				<li><a href="">How Digitax works</a></li>
-				<li><a href="">Digitax guarantees</a></li>
-				<li><a href="">Digitax security and fraud protection</a></li>
-				<li><a href="">Tax forms included with Digitax</a></li>
-				<li><a href="">Digitax en español</a></li>
-				<li><a href="">Self-employed tax center</a></li>
-				<li><a href="">Tax Expert Network</a></li>
-				<li><a href="">Tax reform center</a></li>
-				<li><a href="">Tax Refund Advance</a></li>
-				<li><a href="">E-file Taxes Online</a></li>
-				<li><a href="">Intuit CEO, Sasan Goodarzi</a></li>
-			</ul>
-		</div>
-
-		<div>
-			<p>Online Software Product</p>
-			<ul>
-				<li><a href="">Digitax login</a></li>
-				<li><a href="">Compare Digitax products</a></li>
-				<li><a href="">All online tax preparation software</a></li>
-				<li><a href="">Free Edition tax prep</a></li>
-				<li><a href="">Deluxe tax prep to maximize deductions</a></li>
-				<li><a href="">Premier investment & rental property tax prep</a></li>
-				<li><a href="">Self-employed tax prep</a></li>
-				<li><a href="">Military tax prep discount</a></li>
-				<li><a href="">All Digitax Live Products</a></li>
-				<li><a href="">Digitax Live Basic</a></li>
-				<li><a href="">Digitax Live Deluxe</a></li>
-				<li><a href="">Digitax Live Premier</a></li>
-				<li><a href="">Digitax Live Self-Employed</a></li>
-			</ul>
-		</div>
-
-		<div>
-			<p>CD/Download Products</p>
-			<ul>
-				<li><a href="">Digitax CD/download login</a></li>
-				<li><a href="">Compare CD/download products</a></li>
-				<li><a href="">All CD/download products</a></li>
-				<li><a href="">Check order status</a></li>
-				<li><a href="">Digitax Advantage program</a></li>
-				<li><a href="">Digitax Business for corps</a></li>
-				<li><a href="">Products for previous tax years</a></li>
-				<li><a href="">Digitax CD/download support</a></li>
-			</ul>
-		</div>
+		
 
 		<div>
-			<p>Mobile Tax Apps</p>
+			<p>Digitax</p>
 			<ul>
-				<li><a href="">All Digitax mobile apps</a></li>
-				<li><a href="">Official Digitax mobile app</a></li>
-				<li><a href="">TaxCaster tax calculator</a></li>
-				<li><a href="">ItsDeductible donation tracker app</a></li>
+				<li><a routerLink="/signin">Signin</a></li>
+				<li><a routerLink="/signup">Signup</a></li>
+				<li><a routerLink="/privacy-policy">Privacy Statement</a></li>
+				<li><a routerLink="/terms-service">	Terms of Service</a></li>				
```

Comment:
@Bithika-Redapple You can remove the extra, unnecessary space in front of Terms of Service

Reply:
it is done